### ISSUE TITLE
bin/setup not working on MacOS

### ISSUE THREAD

**Issue by markusguenther** [hedgedoc/hedgedoc#2482]
### Description

Tried to follow the manuall install instructions and failing on the bin/setup step

### Steps to reproduce

1. cloned repository with version it `-b 1.9.3`
2. added slim config file
3. run `bin/setup`

### Expected behaviour

Setup script runs

### Logs

```shell
FATAL: Your Yarn version is not supported.

Please upgrade to version 1.22.0 or higher and try again.
See https://classic.yarnpkg.com/en/docs/install for instructions.
```

### Config

```shell
{
  "production": {
    "db": {
      "dialect": "sqlite",
      "storage": "./db.hedgedoc.sqlite"
   },
    "urlAddPort": true,
    "domain": "localhost"
  }
}
```

### Your Setup

- Host OS: MacOS 12.4
- NodeJS version: 16.15.1
- HedgeDoc version: 1.9.3

**Comment by markusguenther** [comment]
Hope that it is not just my issue :/

<img width="513" alt="Screenshot 2022-07-05 at 14 43 11" src="https://user-images.githubusercontent.com/1014126/177331527-72aa5f31-a1bb-475b-ab9a-f3ff7aa1769e.png">

**Comment by markusguenther** [comment]
Seems that the checks in the setup script does not work with my zsh and the Apple Silicon stuff... I commented out the node and yarn check and then it works.

**Comment by ErikMichelson** [comment]
We can confirm that the setup script currently does not work on MacOS.

**Comment by davidmehren** [comment]
Fixed with #4162
